include/llvm/IR/Value.h

```diff
@@ -199,6 +199,7 @@ class Value {
 
   /// \brief Support for debugging, callable in GDB: V->dump()
   void dump() const;
+  void dumpToFile(const char *path) const; // HLSL Change
```

Comment:
We should also wrap the declaration in `#if !defined(NDEBUG) || defined(LLVM_ENABLE_DUMP)`